Update o365-data-locations.md (#3009)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/o365-data-locations.md

Please review the [Products available by region](https://go.microsoft.com/fwlink/p/?linkid=2093451) page to find data residency information for Microsoft Azure.

> ### Where can I find data residency information for MIcrosoft Bookings?

Reviewer: ```suggestion
### Where can I find data residency information for Microsoft Bookings?
```

> All customer content is stored in the Exchange Online mailbox that hosts the organization's Bookings.

Reviewer: ```suggestion
All customer content is stored in the Exchange Online mailbox that hosts the organization's Microsoft Bookings.
```

> ### Where can I find data residency information for MIcrosoft Shifts?

Reviewer: ```suggestion
### Where can I find data residency information for Microsoft Shifts?
```

> Microsoft Shift is a native app of MS Teams so this adheres to the MS team's data location.

Reviewer: ```suggestion
Microsoft Shifts is a native app of Microsoft Teams so this adheres to the data location of Microsoft Teams.
```

### Where can I find data residency information for Microsoft To Do?

> Microsoft To do Data is stored as tasks in your Exchange Online mailbox, which also hosts data from other Exchange modules such as emails, events, contacts, and/or notes.

Reviewer: ```suggestion
Microsoft To Do data is stored as tasks in your Exchange Online mailbox, which also hosts data from other Exchange modules such as emails, events, contacts, and/or notes.
```
